public/js/lib/editor/index.js

```diff
@@ -121,6 +121,30 @@ export default class Editor {
       },
       'Shift-Ctrl-Backspace': cm => {
         utils.wrapTextWith(this.editor, cm, 'Backspace')
+      },
+      'Ctrl-\\': cm => {
```

Comment:
Is this <kbd>Ctrl</kbd> + <kbd>\\</kbd> ? That will most likely no work on German keyboard cause we don't have <kbd>\\</kbd> as a primary key.

Reply:
All the more reason to switch to a US layout. :) I know what I'm talking about, I grew up with the German layout. It's terrible.

On a more serious note, got any suggestions? I'm happy to switch this to something that's a bit more globally compatible.

Reply:
If you can convince my muscle memory to change. :) If you know all the keys by heart you won't change anytime soon if you not really like change.

What about <kbd>Shift</kbd> + <kbd>Enter</kbd>?

Reply:
OK with me, but that's widely used (Word uses it for a non-paragraph-breaking line break). If my suggestion was too off-Broadway, maybe Shift-Enter is too on-Broadway? <kbd>Ctrl</kbd>+<kbd>.</kbd> maybe?

Reply:
I am not sure about this.